### PR TITLE
SystemUpdateManager API sample

### PR DESCRIPTION
**Pull request by paulmon**

### CODE REVIEW

**Comment by paulmon** [comment]
@saraclay @parameshbabu @bfjelds @gmileka

File: Samples/SystemUpdateManager/CS/MainPage.xaml.cs [hunk 1]
```diff
@@ -0,0 +1,412 @@
...
+        /// </summary>
+        /// <param name="sender"></param>
+        /// <param name="e"></param>
+        private void ActiveHoursBorder_Loaded(object sender, RoutedEventArgs e)
+        {
+            ActiveHoursPopup.VerticalOffset = (SettingsGrid.ActualWidth / 2) - (ActiveHoursBorder.ActualWidth / 2);
+            ActiveHoursPopup.HorizontalOffset = (SettingsGrid.ActualHeight / 2) - (ActiveHoursBorder.ActualHeight / 2);
+        }
+
+        //private void CancelUpdates_Click(object sender, RoutedEventArgs e)
```
Comment (bfjelds) [inline Samples/SystemUpdateManager/CS/MainPage.xaml.cs]: remove commented code?

File: Samples/SystemUpdateManager/CS/README.md [hunk 2]
```diff
@@ -0,0 +1,292 @@
...
+## System Update Manager Overview
+
+![System Update Flow](https://github.com/paulmon/Windows-iotcore-samples/blob/system-update-manager/Resources/images/SystemUpdateManager/SystemUpdateFlow.PNG)
+
+The system update process has the following states:
+
+1. Detect – Determine if any update packages are available.
+2. Download – Download the update packages from the server.
+3. Stage – Unpack the update packages and stage them to be committed later.
+4. Wait for reboot into the Update OS.  The Update OS is an OS that shows gears while the OS changes are being committed.  This is part of the lite servicing stack like what is used on Windows Mobile.  On desktop the flow would be different.
```
Comment (bfjelds) [inline Samples/SystemUpdateManager/CS/README.md]: Maybe: "This part of the lite servicing stack is similar to what is used on Windows Mobile"

File: Samples/SystemUpdateManager/CS/README.md [hunk 3]
```diff
@@ -0,0 +1,292 @@
...
+5. Commit.  Boot into the Update OS which shows gears on IoTCore, in order to commit update package changes.
```
Comment (bfjelds) [inline Samples/SystemUpdateManager/CS/README.md]: you specify "on IotCore" here, is it different elsewhere?  is the specification needed?

File: Samples/SystemUpdateManager/CS/README.md [hunk 4]
```diff
@@ -0,0 +1,292 @@
...
+6. Reboot into updated Main OS.  At this point the update process is complete.
+
+Installing a update package includes staging the update package by extracting the files and settings to a staging area, and then later committing the staged changes. Update packages cannot be rolled back on IoTCore once the commit phase begins.  Uncommitted update packages which have only been staged can be discarded if necessary by cancelling the update.  Once the commit process is started it cannot be interrupted.
```
Comment (bfjelds) [inline Samples/SystemUpdateManager/CS/README.md]: "Installing an update"

File: Samples/SystemUpdateManager/CS/README.md [hunk 5]
```diff
@@ -0,0 +1,292 @@
...
+
+Once the update packages are downloaded and staged the interactive update process may be continued by the developer by rebooting into the update OS and commiting the update packages.  The Update OS is a very small OS that has the sole purpose of committing update packages.  On IoTCore the Update OS shows a screen with moving gears.  Rebooting into the Update OS can be done in response to user input or as part of the business logic of a single purpose device.  A normal system reboot will not change to the UpdateOS, RebootToCompleteInstall() must be called.  Alternatively the developer may choose to wait until the next scheduled automatic update process reboot window as configured by current policies (outside of active hours for example.)
```
Comment (bfjelds) [inline Samples/SystemUpdateManager/CS/README.md]: add comma: "Once the update packages are downloaded and staged, the interactive"

File: Samples/SystemUpdateManager/CS/README.md [hunk 6]
```diff
@@ -0,0 +1,292 @@
...
+            else
+            {
+                Windows.System.Update.SystemUpdateManager.SetFlightRing(newRing);
+            }
+        }
+    }
+
+## Block Automatic Reboots
+
+To tell the update service that the system updates aren’t allowed call **SystemUpdateManager.UnblockAutomaticRebootAsync(id)** where **id** is a unique string composed of numbers letters and dashes. When the critical code execution is completed the code should call **SystemUpdateManager.UnblockAutomaticRebootAsync()** for each **id** that was passed into **BlockAutomaticRebootAsync()**
```
Comment (bfjelds) [inline Samples/SystemUpdateManager/CS/README.md]: add comma: "To tell the udpate service that system updates aren't allowed, call"

File: Samples/SystemUpdateManager/CS/README.md (hunk 2 above)
Comment (paulmon) [reply Samples/SystemUpdateManager/CS/README.md]: reworded

File: Samples/SystemUpdateManager/CS/README.md (hunk 3 above)
Comment (paulmon) [reply Samples/SystemUpdateManager/CS/README.md]: reworded

File: Samples/SystemUpdateManager/CS/README.md (hunk 4 above)
Comment (paulmon) [reply Samples/SystemUpdateManager/CS/README.md]: :)

File: Samples/SystemUpdateManager/CS/README.md (hunk 5 above)
Comment (paulmon) [reply Samples/SystemUpdateManager/CS/README.md]: :)

File: Samples/SystemUpdateManager/CS/README.md (hunk 6 above)
Comment (paulmon) [reply Samples/SystemUpdateManager/CS/README.md]: :)

**Pull request merged**